src/NzbDrone.Core/Download/Clients/Tribler/Models/TriblerDownloadClientApi.cs

```diff
@@ -0,0 +1,257 @@
+namespace NzbDrone.Core.Download.Clients.Tribler
+{
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.5.2.0 (Newtonsoft.Json v11.0.0.0)")]
+    public enum DownloadStatus
+    {
+        [System.Runtime.Serialization.EnumMember(Value = @"WAITING4HASHCHECK")]
+        WAITING4HASHCHECK = 0,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"HASHCHECKING")]
+        HASHCHECKING = 1,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"METADATA")]
+        METADATA = 2,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"DOWNLOADING")]
+        DOWNLOADING = 3,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"SEEDING")]
+        SEEDING = 4,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"STOPPED")]
+        STOPPED = 5,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"ALLOCATING_DISKSPACE")]
+        ALLOCATING_DISKSPACE = 6,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"EXIT_NODES")]
+        EXIT_NODES = 7,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"CIRCUITS")]
+        CIRCUITS = 8,
+
+        [System.Runtime.Serialization.EnumMember(Value = @"STOPPED_ON_ERROR")]
+        STOPPED_ON_ERROR = 9,
+        
+    }
+
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.5.2.0 (Newtonsoft.Json v11.0.0.0)")]
+    public partial class Trackers
```

Comment:
I don't think any of these need to be partial

Reply:
You are correct. I'ts my first time using Newsontoft & c# so i though they were necessary when handling json.